README.md

```diff
@@ -1,3 +1,5 @@
 ## Okta profile browser
+Electron based app to help browse Okta universal directory mappings and profiles.
 
-WIP
+Start the project
```

Comment:
@Ruttala-Lava-Raju  can you also add other commands for testing, budling and debugging ?